README.md

```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
 This is not required though.
 You can generate and use a self-signed certificate.
 
-Here's an example command that generates a 2048-bit RSA certificate valid for a year:
+Here's an example command that generates a 2048-bit RSA certificate valid for a year.
+Before execute command, set shell variables: `$country`, `$state`, `$organization`, `$organization_unit`, `$domain_name`, `$email_address`
 
 ``` bash
-openssl req                     \
-    -x509                       \
-    -nodes                      \
-    -days   365                 \
-    -newkey rsa:2048            \
-    -keyout nginx-proxy-ssl.key \
-    -out    nginx-proxy-ssl.crt
+openssl req                      \
+    -x509                        \
+    -nodes                       \
+    -days 365                    \
+    -sha256                      \
+    -newkey rsa:2048             \
+    -out nginx-proxy-ssl.crt     \
+    -keyout nginx-proxy-ssl.key  \
+    -config <(                   \
+cat <<-EOF
```

Comment:
Instead of creating the file inline, please just include it separately, as a template and tell the user to fill in specific fields. The way it is now it looks more complex than it really has to.

Reply:
Done.